src/System.Net.WebSockets.Client/src/Resources/Strings.resx

```diff
@@ -121,4 +121,10 @@
   <data name="net_securityprotocolnotsupported" xml:space="preserve">
     <value>The requested security protocol is not supported.</value>
   </data>
+  <data name="net_WebSockets_UWPClientCertSupportRequiresWindows10InsiderPreviewBuild16215OrGreater" xml:space="preserve">
+    <value>Support for client certificates in UWP requires Windows 10 Insider Preview Build 16215 or greater.</value>
```

Comment:
Then again, we can keep this as-is for now and update it later when RS3 RTMs. That may or may not be within the 2.1.0 milestone. We should open an issue to track it.

---
In reply to: [126186269](https://github.com/dotnet/corefx/pull/21908#discussion_r126186269) [](ancestors = 126186269,126160835)